code/modules/antagonists/eldritch_cult/eldritch_book.dm

```diff
@@ -149,5 +175,66 @@
 	icon_state = initial(icon_state)
 	return ..()
 
-/obj/item/forbidden_book/debug
-	charge = 100
+/datum/brain_trauma/fascination
+	name = "Delirium"
+	desc = "Patient is deluded into believing that omnipotent extraterestrial entities meddle in our world."
+	scan_desc = "lovecraftian madness"
+	gain_text = "Ah! A greater purpose."
+	lose_text = "You come to the realization that there are no omnipotent Gods that can save you from the monotony of your day-to-day job."
+	resilience = TRAUMA_RESILIENCE_SURGERY
+
+/datum/brain_trauma/fascination/on_gain()
+	message_admins("[ADMIN_LOOKUPFLW(owner)] has become fascinated.")	//self antag warning?
+	log_game("[key_name(owner)] has become fascinated.")
+
```

Comment:
I feel like you should add antag datum to fascinated people so admins can easily see who is fascinated at the time.

Reply:
Hmm. But are they really antag?

Reply:
If the stuff they are doing would be considered self antag  if they were doing it as a non antag they should have an antag datum